app/strings.php

```diff
@@ -0,0 +1,1365 @@
+<?php
+
+// In order to support early translations tasks, this file contains the strings for features/pages that have yet to be
+// implemented.
+
+return [
+    // Home Page - https://github.com/accessibility-exchange/platform/issues/750
+    '0.0 Home Page' => [
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A3399
+        'HomePage' => [
+            __('Contrast'),
+            __('Back to top'),
+            __('About'),
+            __('Glossary'),
+            __('Privacy Policy'),
+            __('Available :start to :end Easter Standard Time'),
+            __('Available :start to :end :timeZone'),
```

Comment:
Added two examples as I'm not sure if the Time Zone will be interpolated.

Reply:
We'll want the interpolated version.